md/languages.md

```diff
@@ -15,33 +15,33 @@ Officially supported language means that:
 
 ## Select BPM Portal user interface language
 
-For the web desktop version: go to the top right corner, click on _**Settings**_/_**Language**_ and select a language from the list.
+For the web desktop version:
+1. Go to the top right corner and click on _**Settings**_/_**Language**_
+1. Select a language from the drop-down list.
 
 For the web mobile version:
 
 1. Click on the _**Tools icon**_
-2. Click on _**Languages**_
-3. Select a new language from the drop-down list
+1. Click on _**Languages**_
+1. Select a new language from the drop-down list
 
 ## Add a language to Bonita BPM Portal
 
-::: warning
-We are currently updating the translation process. Among other things this update impacts the Crowdin project structure. Instruction below might be temporarily inaccurate.
-:::
-
 ::: info
 Instructions below assume that the language you want to add is already available in the community translation project. If translation is not available see [Translate Bonita BPM Portal](#Translate-Bonita-BPM-Portal) for instruction about how to collaborate to a new or ongoing translation.
```

Comment:
No I try to insert a link to an anchor but forget to declare the anchor (I use GitHub syntax instead and doesn't seems to work).